packages/grafana-ui/src/components/Table/Table.mdx

```diff
@@ -7,11 +7,11 @@ Used for displaying tabular data
 
 ## Sub-tables
 
-Sub-tables are supported through the usage of the prop `subData` Dataframe array.
-The frames are linked to each row using the following custom properties under `dataframe.meta.custom`
+Sub-tables are supported by adding `FieldType.nestedFrames` to the field that contains the nested data in your dataframe.
 
-- **parentRowIndex**: number - The index of the parent row in the main dataframe (under the `data` prop of the Table component)
-- **noHeader**: boolean - Sets the noHeader of each sub-table
+This nested fields values can contain one or more dataframes. Each of these dataframes will be rendered as one unique sub table.
```

Comment:
From the code it seems like it always has to be an array even if it's a single frame. Can we change it to allow just a single DataFrame?